docs/keystone.smartcity.conf

```diff
@@ -35,4 +35,15 @@ user_id_attribute = uid
 user_name_attribute = cn
 user_mail_attribute = mail
 
-debug_level = 4096
+# debug_level = 4095
```

Comment:
Leftover?

Reply:
Fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/146/commits/262c474abd85fdcb3cfc89b9b85138724b887ca6